### ISSUE TITLE
Source map reference not updated

### ISSUE THREAD

**Issue by adrienkohlbecker** [smysnk/gulp-rev-all#60]
Hello @smysnk 

I'm having the same issue as in #53 but with the sourcemap reference :

```
/*# sourceMappingURL=application.css.map */
//# sourceMappingURL=application.js.map
```

It does not get updated with the renamed map file

**Comment by smysnk** [comment]
Should be fixed in 0c0bb83.

**Comment by adrienkohlbecker** [comment]
Great thanks, I'll check it out
